src/containers/Tablets/Tablets.tsx

```diff
@@ -32,11 +32,7 @@ const columns: DataTableColumn<TTabletStateInfo & {fqdn?: string}>[] = [
             return i18n('Type');
         },
         render: ({row}) => {
-            return (
-                <span>
-                    {row.Type} {row.Leader ? <Text color="secondary">leader</Text> : ''}
-                </span>
-            );
+            return <span>{row.Type}</span>;
```

Comment:
Tablet isn't a leader by default. If `row.Leader === false`, there should be `follower` text. `leader` text is unneeded, because it's true in most of the cases